scripts/build.sh

```diff
@@ -3,4 +3,4 @@
 # This is required because it is not possible to both inlcude .js
```

Comment:
Time to remove these obsolete comments?

Reply:
Removed the build script entirely as it's now useless